frontend/src/metabase/setup/actions.ts

```diff
@@ -62,14 +59,9 @@ export const loadDefaults = createAsyncThunk<void, void, ThunkConfig>(
   },
 );
 
-export const SELECT_STEP = "metabase/setup/SUBMIT_WELCOME_STEP";
+export const SELECT_STEP = "metabase/setup/SELECT_STEP";
```

Comment:
I'm assuming this was always supposed to be `"SELECT_STEP"`, not `"SELECT_WELCOME_STEP"` which was assigned to `SUBMIT_WELCOME` on the deleted line 68 below